locations/spiders/dairy_queen_us.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.dict_parser import DictParser
+
+
+class DairyQueenUSSpider(Spider):
+    name = "dairy_queen_us"
+    allowed_domains = ["prod-dairyqueen.dotcmscloud.com"]
+    start_urls = ["https://prod-dairyqueen.dotcmscloud.com/api/es/search"]
+    custom_settings = {"ROBOTSTXT_OBEY": False}  # Missing robots.txt
+    brands = {
+        "Food and Treat": {
+            "brand": "DQ Grill & Chill",
+            "brand_wikidata": "Q1141226",
+            "extras": {"cuisine": "ice_cream;burger"},
+        },
+        "Treat Only": {"brand": "Dairy Queen", "brand_wikidata": "Q1141226", "extras": {"cuisine": "ice_cream"}},
+    }
+
+    def start_requests(self):
+        yield JsonRequest(
+            url=self.start_urls[0],
+            method="POST",
+            headers={
+                "Referer": "https://www.dairyqueen.com/",
+            },
+            data={"size": 10000, "query": {"bool": {"must": [{"term": {"contenttype": "locationDetail"}}]}}},
+        )
+
+    def parse(self, response):
+        for location in response.json()["contentlets"]:
+            item = DictParser.parse(location)
+            item["lat"], item["lon"] = location.get("latlong", ",").split(",", 2)
+            item.update(self.brands[location["conceptType"]])
```

Comment:
Updating extras in this way wont work if we ever change DictParser to collect an extra, like fax address or twitter. Or like LinkedDataParser when started parsing hotel extras (stars).

Reply:
Fixed in https://github.com/alltheplaces/alltheplaces/pull/6298 (I hope)